src/website/views/components/TimestampFilter/TimestampFilter.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import Accordion from '@mui/material/Accordion';
+import AccordionSummary from '@mui/material/AccordionSummary';
+import AccordionDetails from '@mui/material/AccordionDetails';
+import Typography from '@mui/material/Typography';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { Button, Grid } from '@mui/material';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
+import styles from './timestampfilter.module.css';
+
+// FIX ELEVATION ISSUE !!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+function TimestampFilter(props) {
+  const [startDate, setStartDate] = useState('');
+  const [endDate, setEndDate] = useState('');
+
+  const dispatch = props.dispatch;
+
+  const handleApplyChange = () => {
+    let cond1 = startDate != null && endDate != null;
+    let cond2 = _parseISOString(endDate) > _parseISOString(startDate);
```

Comment:
Could we have these variables `cond1` and `cond2` be more descriptive? 

Also, I like the constraint you added for `cond2` :) we should return some kind of error message in the case where this is false. We can talk about this in the next review iteration